src/conditionals/third-party/wordproof-core-inactive.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Why is this file called `wordproof-core-...` as opposed to just `wordproof-...`?

Reply:
Mostly to differentiate between the WordProof standalone plugin and this integration. I propose to use wordproof-plugin-inactive instead.